foreman.te

```diff
@@ -293,10 +299,20 @@ optional_policy(`
 
 tunable_policy(`passenger_can_connect_openstack',`
     ifdef(`distro_rhel6', `
+        # keystone (identity service)
         corenet_tcp_connect_commplex_port(passenger_t)
+        # nova (compute service)
+        # via foreman_allowed_port_t
```

Comment:
If you disable `passenger_can_connect_openstack` on EL6, it can still connect to Nova.